components/form/EmployeeForm.tsx

```diff
@@ -67,6 +68,42 @@ const EmployeeForm = ({ onSubmit, onCancel, employee }: EmployeeFormProps) => {
         }
     }, [employeeFields])
 
+    const onConfirm = (event: React.MouseEvent) => {
+        event.preventDefault()
+        try {
+            onSubmit(validateEmployeeFields(employeeFields))
+        } catch (error) {
+            errorHandler(error as Error)
```

Comment:
Not really an issue of this pr but is the casting really needed if the errorHandler already accepts that as type? What happens without the type cast?

Reply:
![epoc_errorhandling_unknown](https://user-images.githubusercontent.com/110386486/217783987-251a109d-3ac2-4610-919d-68e6bea64a2a.png)